### ISSUE TITLE
Please consider accessibility at least when rendering the manifesto

### ISSUE THREAD

**Issue by mleadbeater** [reactivemanifesto/reactivemanifesto#142]
The rendered copy (web and pdf) use a particularly low contrast font setting. Granted the Reactive Manifesto doesn't mention the idea that a system should be accessible but I feel that increasing the readability of the website would be a great service to your members and potential members. Reference 2, below gives a decent explanation of the concepts in 1.

References:
1. https://www.w3.org/TR/WCAG21/#contrast-minimum
2. https://webaim.org/articles/contrast/

**Comment by dtonhofer** [comment]
Absolutely. As you get older, eyesight declines. Grey on white is not the best choice.
